package.json

```diff
@@ -117,6 +120,12 @@
     "validateSelfClosingTags": true,
     "validateTemplateString": true
   },
+  "stylelint": {
+    "root": true,
```

Comment:
The [documentation](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md) doesn't mention `root`, what does this do?

Reply:
I personally didn't know. It was generated for me. The eslintConfig configuration section also has the root. Maybe it has something to do with vue-cli? I didn't find any document as well. https://github.com/vuejs/vue-cli/blob/1a8394434ab074d6d15412f17d8374e84cee9314/packages/%40vue/cli/lib/Generator.js#L99

Reply:
I just read through the source of `@ascendancyy/vue-cli-plugin-stylelint`, and it seems pretty rough, and IMO unnecessary. It's a whole lot of code that does almost nothing beyond this:

`stylelint src/**/*.vue`

So my preference would be to drop `@ascendancyy/vue-cli-plugin-stylelint` and do

`"lint:style": "stylelint 'src/**/*.vue'"`

If we anticipate including CSS in files aside from just vue single file components, we can add more extensions to that glob expression.

Reply:
If we cared about linting on build, I'd say it's worth using the plugin, but I'm not a fan of linters causing build errors.

Reply:
Sounds good, I'll give it a try and see if everything works well without it.

Reply:
Everything works great, include the autofix that I would use. I'll push the change.

Reply:
Hi @zachmullen, this PR is ready for further review